alot/db/utils.py

```diff
@@ -110,7 +110,7 @@ def message_from_file(handle):
 
     # handle OpenPGP signed data
     if (m.is_multipart() and
-        m.get_content_subtype() == 'signed' and
+            m.get_content_subtype() == 'signed' and
```

Comment:
this is one of those wierd corners of PEP8, if you indent 4 spaces after an `if` you end up with this:
```python
if (foo and
    bar):
    print 'yes!'
```
PEP8 (https://www.python.org/dev/peps/pep-0008/#indentation) chooses not to wade into the mess too much, and says either make them flat as above, or push the second line of the if statement in and additional 4 spaces (total of 8). Personally, I think they're both ugly, btu the patch is much more readable than the example in this comment. Applies to the next comment of yours too.